product_customer_code_sale/__openerp__.py

```diff
@@ -1,44 +1,21 @@
 # -*- coding: utf-8 -*-
-#
-#
-#    Copyright (C) 2013 Agile Business Group sagl (<http://www.agilebg.com>)
-#    Author: Nicola Malcontenti <nicola.malcontenti@agilebg.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as published
-#    by the Free Software Foundation, either version 3 of the License, or
-#    (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-#
+# Copyright 2016 Nicola Malcontenti - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Product Customer code on sale",
-    "version": "1.0",
+    "version": "8.0.1.0.0",
     "author": "Agile Business Group,Odoo Community Association (OCA)",
     "website": "http://www.agilebg.com",
     "category": "Sales Management",
+    'license': 'AGPL-3',
     "depends": [
-        'base',
         'product',
         'sale',
         'product_customer_code'
```

Comment:
this one is in product-attributes which I don't see referenced in the oca-dependencies.txt file. Can you fix this?